packages/utils/src/misc.ts

```diff
@@ -372,6 +372,15 @@ export const crossPlatformPerformance: Pick<Performance, 'now' | 'timeOrigin'> =
   }
 
   if (getGlobalObject<Window>().performance) {
+    // tslint:disable-next-line:deprecation
+    if (!performance.timing) {
+      return performanceFallback;
+    }
+    // tslint:disable-next-line:deprecation
+    if (!performance.timing.navigationStart) {
+      return performanceFallback;
+    }
+
```

Comment:
I think this is too early? We may have access to `performance.timeOrigin` and that's our preferred value to use.